__tests__/mocks/dprNewApplicationFactory.test.ts

```diff
@@ -15,12 +15,13 @@
  * along with Digital Planning Register. If not, see <https://www.gnu.org/licenses/>.
  */
 
-import { DprApplication } from "@/types";
 import {
   PostSubmissionAssessment,
   PriorApprovalAssessment,
 } from "@/types/odp-types/schemas/postSubmissionApplication/data/Assessment";
 import {
+  checkConsultationDates,
+  checkConsultationInProgress,
```

Comment:
these methods should live inside the test as they only really relate to this file (and aren't mocks) :)